template.tpl

```diff
@@ -269,7 +280,9 @@ const onFailure = () => {
   data.gtmOnFailure();
 };
 
-injectScript(initScriptUrl, onSuccess, onFailure, 'bizrate');
+let cacheToken = data.spa ? null : 'bizrate';
```

Comment:
🟢 _(nit)_: Since `cacheToken` is not going to be reassigned, we should probably use `const` here to reduce mutability.

Reply:
Good catch!